WhaleSpotting/react-app/src/styles/Profile.scss

```
@@ -0,0 +1,133 @@
+@import './Constants.scss';
+
+.body {
+    max-width: 1024px;
+    font-family: $fontFamily;
+    margin: 0 auto 0 auto;
```

```suggestion
    margin: 0 auto;
```
This can be simplified slightly :)